Python/Product/Analysis/Parsing/Ast/MemberExpression.cs

```diff
@@ -76,7 +76,11 @@ public sealed class MemberExpression : Expression {
             res.Append('.');
             if (!this.IsIncompleteNode(ast)) {
                 res.Append(this.GetSecondWhiteSpaceDefaultNull(ast));
-                res.Append(this.GetVerbatimImage(ast) ?? _name);
+                if (format.UseVerbatimImage) {
+                    res.Append(this.GetVerbatimImage(ast) ?? _name);
```

Comment:
But we are round tripping :)  I think getting the private prefix first, and then getting the AST with the private prefix will give us the expression w/ the mangled identifiers.  And it's what we're already doing elsewhere.

---
In reply to: [64830525](https://github.com/Microsoft/PTVS/pull/1319#discussion_r64830525) [](ancestors = 64830525)